src/networkremote/networkremote.h

```diff
@@ -0,0 +1,45 @@
+#ifndef NETWORKREMOTE_H
+#define NETWORKREMOTE_H
+
+#include <QObject>
+#include <QTcpServer>
+#include <QTcpSocket>
+#include <QHostAddress>
+#include <QSettings>
+
+#include "tcpserver.h"
+#include "networkremote/remotesettings.h"
+
+class Application;
+class QThread;
+
+class NetworkRemote : public QObject
+{
+     Q_OBJECT
+public:
+    static const char* kSettingsGroup;
+    explicit NetworkRemote(Application* app, QObject *parent = nullptr);
+    static NetworkRemote* Instance();
+    ~NetworkRemote() override;
+
+public Q_SLOTS:
+  void Init();
+  void Update();
+  void LoadSettings();
+  void startTcpServer();
+  void stopTcpServer();
+
+private:
+  Application *app_;
+  bool use_remote_;
+  bool local_only_;
+  int remote_port_;
+  QHostAddress ipAddr_;
+  TcpServer *server_;
+  QThread *original_thread_;
```

Comment:
never used

Reply:
remote_port_ is used in networkremote.cpp

Reply:
original_thread_ is not used, that's the line I commented on

Reply:
Removed

Reply:
Its still here

Reply:
Removed